### ISSUE TITLE
Blue text on blue background on home page fails accessibility standards

### ISSUE THREAD

**Issue by nikiforosk** [opentargets/webapp#114]
I used a contrast checker to test the blue text on the blue background on the home page:
http://webaim.org/resources/contrastchecker/

![home-screen shot 2017-03-27 at 11 54 54](https://cloud.githubusercontent.com/assets/20280139/24912839/39ab3990-1ec7-11e7-83d7-bb27cf9dc8dd.png)

The text in question (e.g. in BRAF) is "normal text" (14px). 
 
This colour combination fails both standards for normal text:

Foreground: b2def9 (light blue)
Background: 0f74c2 (darker blue)
Contrast Ratio: 3.42:1

Normal Text
WCAG AA: **Fail**
WCAG AAA: **Fail**

Large Text
WCAG AA: Pass
WCAG AAA: Fail

Level AA requires a contrast ratio of 4.5:1 for normal text and 3:1 for large text. 
Level AAA requires a contrast ratio of 7:1 for normal text and 4.5:1 for large text. 

![screen shot 2017-04-11 at 14 53 38](https://cloud.githubusercontent.com/assets/20280139/24912951/8d2139da-1ec7-11e7-8efd-72bec2f687bd.png)

This means that some people won't be able to read the blue text on the blue background.

**Comment by nikiforosk** [comment]
It's worth noting that even white on blue fails the AAA standard for normal text:

Foreground: ffffff (white)
Background: 0f74c2 (darker blue)
Contrast Ratio: 4.89:1

Normal Text
WCAG AA: Pass
WCAG AAA: **Fail**

Large Text
WCAG AA: Pass
WCAG AAA: Pass

![screen shot 2017-04-11 at 14 53 07](https://cloud.githubusercontent.com/assets/20280139/24913096/f79c12e4-1ec7-11e7-9bef-ff3d4ae68618.png)

**Comment by nikiforosk** [comment]
I also find it inconsistent that we use white for "Try" and blue for the links to "BRAF", "PTEN, etc. 
But on the line below the hyperlinked text is white and the rest of the text blue...

![home-screen shot 2017-03-27 at 11 54 54](https://cloud.githubusercontent.com/assets/20280139/24912839/39ab3990-1ec7-11e7-83d7-bb27cf9dc8dd.png)

**Comment by emepyc** [comment]
They are not the same thing, so I don't understand what you mean by "inconsistent".
In the upper line, you can use the search without clicking on any of the examples, "Try" is in white to give a bit of highlight that you can try any of the alternatives you have there.
In the line below, you access to a functionality that is not possible without following the link, so the highlight is in how to access the batch search.

For me, the important bit is what you highlight, where you want to put the accent, rather than if in the upper line it is the first part and in the other is the second.

Under that definition of "consistency", the notifications bell is inconsistent because it is the only thing red in the page.

**Comment by emepyc** [comment]
Regarding the contrast, you mention that: "It's worth noting that even white on dark blue fails the AAA standard for normal text". 
Do you mean that "some people won't be able to read the blue text on the blue background."?
I find this surprising.

The analytics show a high proportion of users trying the examples ("BRAF" being the most used example), so not sure if the contrast is creating any problem here.

**Comment by nikiforosk** [comment]
It's inconsistent because on the first line the hyperlinked text is blue and the non-clickable text is white, while on the second line it's the other way around. 

The problem is that you can't tell whether a text is clickable or not based on how it is formatted in two successive sentencers. Imagine a text in which the hyperlinked text in the first sentence is underlined but on the second sentence the underlined text is not hyperlinked. 

If you want to keep the highlight effect on the clickable text, you can use blue for "Try" on the first line and white for the links of the examples (which will also make them more readable).

**Comment by nikiforosk** [comment]
Regarding the AAA standard on white on blue: Yes, this means that for some people this is hard to read. I can't tell how many these people are (and probably also depends on the quality of their screens). But given that our users are predominantly in their late 30s or even older (which is when your sight starts deteriorating) it could be a problem.

The analytics can't tell us whether people can read the text. The standards have been developed as a warning for that purpose and I suggest that we conform to them.

**Comment by emepyc** [comment]
As I said before, the links in the upper line are completely optional, it doesn't matter if the user click on those links, they can always type "BRAF" in the search box (and as I said above, their are using the examples based on our analytics) (they didn't look hyperlinks in the first place!).

In the second case we don't want the users to miss that there is a new functionality and the link is highlighted. If the users find it confusing, we can change it.

I'm not saying the current version is perfect, I'm saying that functionality should prevail over forced UI consistency.
I'm happy for you to tweak the CSS and play with it. If you find a solution that woks better I'd be happy to push it.

**Comment by nikiforosk** [comment]
Luca and I explored a few alternatives. This is our preferred solution:
![home-new-certificate-screen shot 2017-04-19 at 10 42 07](https://cloud.githubusercontent.com/assets/20280139/25183062/6c5c7f1e-250e-11e7-8f13-5977fd8c7567.png)

- White instead of blue font for higher contrast
- Consistent style for hyperlinks in this section (dotted line as in our [documentation pages](https://www.targetvalidation.org/faq))
- Decorated "New" and simpler text to draw attention to new feature

The other designs that we considered are here:
https://drive.google.com/drive/folders/0B1nylrXelBI8N2dNUFJiYlh1TE0?usp=sharing

**Comment by emepyc** [comment]
If we make this change, those will be the only links with that styling. The problem will arise again when we revisit/create our pattern library.

**Comment by peatroot** [comment]
No users have complained about this. Discussed with @andrewhercules.
